skatepark/styles/green-pink.json

```
@@ -0,0 +1,28 @@
+{
+	"$schema": "https://json.schemastore.org/theme-v1.json",
+	"version": 2,
+	"title": "Green Pink",
```

```suggestion
	"title": "Green pink",
```

_Changing to sentence case to match the "Blue cream" name above._ 🙂